lib/openhab/dsl/types/point_type.rb

```diff
@@ -16,7 +16,14 @@ class PointType
         # @param latitude [DecimalType, QuantityType, StringType, Numeric]
         # @param longitude [DecimalType, QuantityType, StringType, Numeric]
         # @param altitude [DecimalType, QuantityType, StringType, Numeric]
-        def initialize(*args) # rubocop:disable Metrics
+        # @param :latitude [DecimalType, QuantityType, StringType, Numeric]
+        # @param :lat [DecimalType, QuantityType, StringType, Numeric]
+        # @param :longitude [DecimalType, QuantityType, StringType, Numeric]
+        # @param :long [DecimalType, QuantityType, StringType, Numeric]
+        # @param :altitude [DecimalType, QuantityType, StringType, Numeric]
+        # @param :alt [DecimalType, QuantityType, StringType, Numeric]
+        def initialize(*args, **kargs) # rubocop:disable Metrics
```

Comment:
Normally you use "kwargs", not "kargs".  Though I'm also wondering if you should name them or not (not being using a splat as you have). Or even if we should use kwargs or only accept an _actual_ hash (which should functionally be the same from the caller's point of view, though there are land mines in Ruby 2.6, 2.7, and 3.0 with hash-as-last-arg-in-combination-with-splats-and-kwarg-splats). I think I'm honestly leaning that way, given how we're expecting it to be an atomic hash with _either_ one set of opts _or_ a different set of opts, instead of being several optional modifying params, each of them optional in their own right.